src/main/java/dto/order/OrderCreateRequestShippingAddressDto.java

```diff
@@ -0,0 +1,10 @@
+package dto.order;
+
+import javax.validation.constraints.NotNull;
+import lombok.Data;
+
+@Data
+public class OrderCreateRequestShippingAddressDto {
+    @NotNull
```

Comment:
```suggestion
    @NotBlank
```
And validate maximum length too.